dashboard/app/assets/stylesheets/application.scss

```diff
@@ -317,6 +317,7 @@ img.video_thumbnail {
       margin-top: 2px;
       font-size: 12px;
       display: inline-block;
+      border-radius: 4px;
```

Comment:
I feel like we generally use 4px as our default border radius these kinds of text boxes or buttons. Maybe something we can consider standardizing at some point? cc: @levadadenys